scripts/scil_shuffle_streamlines.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import random
+
+from dipy.io.stateful_tractogram import Space, StatefulTractogram
+from dipy.io.streamline import save_tractogram
+
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_overwrite_arg, add_reference_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+
+
+def _build_args_parser():
+
+    p = argparse.ArgumentParser(
+        formatter_class=argparse.RawTextHelpFormatter,
+        description='Shuffle the ordering of streamlines.')
+
+    p.add_argument('in_tractogram',
+                   help='Input tractography file.')
+    p.add_argument('out_tractogram',
+                   help='Output tractography file.')
+    p.add_argument('--seed', type=int, default=None,
+                   help='Random number generator seed [%(default)s].')
+    add_reference_arg(p)
+    add_overwrite_arg(p)
+
+    return p
+
+
+def main():
+    parser = _build_args_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, args.in_tractogram)
+    assert_outputs_exist(parser, args, args.out_tractogram)
+
+    sft = load_tractogram_with_reference(parser, args, args.in_tractogram)
+    streamlines = list(sft.get_streamlines_copy())
+    random.shuffle(streamlines, random=args.seed)
+
+    smoothed_sft = StatefulTractogram(streamlines, sft, Space.RASMM)
```

Comment:
Is the smoothing part missing ?

Reply:
That would be in the other script, scil_smooth_streamlines.py

Reply:
It really is kind of nitpicky from my part and somewhat unnecessary, but since the tractogram is not smoothed, only shuffled in this script, could the variable be renamed to something lilke `shuffled_sft` to avoid potential confusion ?

Reply:
Ahh ok, I didn't understand the question. I thought you said that because the name of the PR start with smooth and you were confused.

And I was the one confuse, 100% my bad. Sorry, you are right.